packages/netlify-cms-core/src/actions/search.ts

```diff
@@ -28,51 +28,32 @@ export function searchingEntries(searchTerm: string, searchCollections: string[]
   return {
     type: SEARCH_ENTRIES_REQUEST,
     payload: { searchTerm, searchCollections, page },
-  };
+  } as const;
 }
 
-export function searchSuccess(
-  searchTerm: string,
-  searchCollections: string[],
-  entries: EntryValue[],
-  page: number,
-) {
+export function searchSuccess(entries: EntryValue[], page: number) {
   return {
     type: SEARCH_ENTRIES_SUCCESS,
     payload: {
-      searchTerm,
-      searchCollections,
```

Comment:
Couldn't figure out why do we need to send the same data. If it's really needed, I can bring it back.